  Reason: disable xf86config generation
Upstream: http://sourceforge.net/tracker/index.php?func=detail&aid=2817192&group_id=69596&atid=525126
  Source: https://bugs.gentoo.org/276119 , Jonathan Dahan patched to make the default no

diff --git a/configure.in b/configure.in
index a4f5fbb..ccf6e9f 100644
--- a/configure.in
+++ b/configure.in
@@ -585,19 +585,27 @@ 
-if test "$WCM_XORG_XSERVER_1_6" == yes; then
-	AC_MSG_CHECKING(for lib xf86config)
-	if test -f $WCM_XLIB_DIR/libxf86config.a -o -f $WCM_XLIB_DIR/libxf86config.so; then
-		if test -f $WCM_XFREE86_DIR/$XFREE86SUBDIR/xf86Parser.h; then
-			AC_MSG_RESULT([found, $WCM_XFREE86_DIR/$XFREE86SUBDIR])
-			WCM_ENV_XF86CONFIG=yes
-			CFLAGS="$CFLAGS -I$WCM_XFREE86_DIR/$XFREE86SUBDIR"
-		elif test -f $WCM_XORGSDK_DIR/xf86Parser.h; then
-			AC_MSG_RESULT([found, $WCM_XORGSDK_DIR])
-			WCM_ENV_XF86CONFIG=yes
-		elif test -f $WCM_XLIB_DIR/../include/xf86Parser.h; then
-			AC_MSG_RESULT([found, $WCM_XLIB_DIR/../include])
-			WCM_ENV_XF86CONFIG=yes
-			CFLAGS="$CFLAGS -I$WCM_XLIB_DIR/../include"	

+WCM_OPTION_XF86CONFIG=no
+AC_ARG_ENABLE(xf86config,
+        AC_HELP_STRING([--enable-xf86config], [Enable libxf86config support if present [[default=no]]]),
+        [ WCM_OPTION_XF86CONFIG=$enableval ],
+        [ WCM_OPTION_XF86CONFIG=no ])
+
+if test "$WCM_OPTION_XF86CONFIG" = yes; then
+	AC_MSG_CHECKING(for lib xf86config)
+	if test -f $WCM_XLIB_DIR/libxf86config.a -o -f $WCM_XLIB_DIR/libxf86config.so; then
+		if test -f $WCM_XFREE86_DIR/$XFREE86SUBDIR/xf86Parser.h; then
+			AC_MSG_RESULT([found, $WCM_XFREE86_DIR/$XFREE86SUBDIR])
+			WCM_ENV_XF86CONFIG=yes
+			CFLAGS="$CFLAGS -I$WCM_XFREE86_DIR/$XFREE86SUBDIR"
+		elif test -f $WCM_XORGSDK_DIR/xf86Parser.h; then
+			AC_MSG_RESULT([found, $WCM_XORGSDK_DIR])
+			WCM_ENV_XF86CONFIG=yes
+		elif test -f $WCM_XLIB_DIR/../include/xf86Parser.h; then
+			AC_MSG_RESULT([found, $WCM_XLIB_DIR/../include])
+			WCM_ENV_XF86CONFIG=yes
+			CFLAGS="$CFLAGS -I$WCM_XLIB_DIR/../include"	
+		fi
 	fi
 fi
 
